include/megaapi.h

```diff
@@ -71,6 +71,7 @@ class MegaContactRequestList;
 class MegaShareList;
 class MegaTransferList;
 class MegaApi;
+class MegaClient;
```

Comment:
`MegaClient` belongs to the low level API. It can't be used in the public interface of the intermediate layer.